src/test/java/com/saucelabs/kv004/ui/AddAnonProblemTest.java

```diff
@@ -0,0 +1,91 @@
+package com.saucelabs.kv004.ui;
+
+import com.saucelabs.pages.AdminPage;
+import com.saucelabs.pages.AnyPage;
+import com.saucelabs.pages.ProblemPage;
+import com.saucelabs.utility.Constant;
+import com.saucelabs.utility.SingletonWebDriver;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by vika on 02.09.15.
+ * First test class adds problem as anon
+ * Second test class approve problem as admin.
+ *
+ */
+public class AddAnonProblemTest {
+
+    AnyPage anyPage;
+    ProblemPage problemPage;
+    AdminPage adminPage;
+    WebDriver driver = SingletonWebDriver.getInstance();
+
+    //One @DataProvider for both test classes
+    @DataProvider(name = "TestData", parallel = false)
+    public static Object[][] testDataExample() {
+        return new Object[][]{
+                new Object[]{
+                        "admin@.com",
+                        "admin",
+                        "test problem name",
```

Comment:
you can take "admin@.com", and password from Constant class in utility package. And also please create several problems and delete/approve them.